src/components/common/BcRegContacts.vue

```diff
@@ -24,14 +24,14 @@ export default class BcRegContacts extends Vue {
     {
       icon: 'mdi-phone',
       key: 'Canada &amp; U.S. Toll Free:',
-      val: '1-877-370-1033',
-      href: 'tel:+1-877-370-1033'
+      val: '1-877-526-1526',
+      href: 'tel:+1-877-526-1526'
     },
     {
       icon: 'mdi-phone',
       key: 'Victoria Office:',
-      val: '250-370-1033',
-      href: 'tel:+1-250-370-1033'
+      val: '250-387-7848',
+      href: 'tel:+1-250-387-7848'
```

Comment:
Does this change need to be done in Filings UI, Create UI, Auth UI, Namerequest UI, Registry UI?

Reply:
Create UI already has this change.
PPR UI already has this change.

Reply:
Well actually, Create has this Canada phone number but a different Victoria number 🤷‍♂️ 

```
    <li class="contact-container">
      <v-icon class="contact-icon" size="18px">mdi-phone</v-icon>
      <span class="px-2">Victoria Office:</span>
      <a href="tel:+1-250-952-0568" class="contact-value">250-952-0568</a>
    </li>
```

Reply:
Interesting. Designs call for these specifically 🤔

Reply:
https://projects.invisionapp.com/d/main/#/console/21805570/463592649/preview

Reply:
What is the number in Create, i could double check with design on this one for sure

Reply:
If Create UI is different for the Contact # in step 2, it might need to be updated, potentially. 
Yui shared designs for Step 2 contact: https://invis.io/3511UPXPMT2P#/463492960_Step_2G-help_DBA
Which matches the designs linked above for Edit UI

Reply:
I've put the onus on Yui to note any required changes somewhere.

Reply:
Copy that, sounds good 👍

Reply:
Yui responded:

"_The contact information is contextual. Sometimes we show Victoria number and sometimes not. Sometimes the BC Registries number is different depending on whether we want to direct the calls._

_If the designs coming in have different numbers, that’s a Linda/Sienna question for sure. Though, I doubt even they have a master list of which filing’s helps are supposed to go where._

_**With SPGP, there was no intention to introduce new phone numbers**. So this is worthwhile flagging if you see discrepancies between what code we’re reusing vs the design._"

^^ The emphasis is mine. Recommend you go ahead and double-check what numbers to use here.